common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -5099,9 +5099,15 @@ schematic = (function() {
 	}
 
 	ConnectionPoint.prototype.toString = function() {
-	    return '<ConnectionPoint ('+this.offset_x+','+this.offset_y+') '+this.parent.toString()+'>';
+	    return edx.StringUtils.interpolate('<ConnectionPoint ({offset_x}, {offset_y}) {parent}>',
+	    {
+		offset_x: this.offset_x,
+		offset_y: this.offset_y,
+		parent: this.parent.toString()
```

Comment:
What is parent? Could it include tags?

Reply:
Parent is Component the base class of all other entities defined here including ConnectionPoint.  The base class doesn't override toString() directly.  Specifications are covered by this change.

Reply:
If I understand correctly, you’ll want:
```
parent: edx.StringUtils.HTML(this.parent.toString())
```
So that this doesn’t get HTML-escaped.

Reply:
I've made this change.  @robrap Is this mergable from your point of view?